pkg/load/load.go

```diff
@@ -124,7 +124,7 @@ func Config(path, unresolvedPath, registryPath string, info *ResolverInfo) (*api
 
 	switch {
 	case len(path) > 0:
-		data, err := ioutil.ReadFile(path)
+		data, err := config.ReadFileMaybeGZIP(path)
```

Comment:
Yeah I agree, this is unlikely to be enough. This isn't even everything in this package:
```
pkg/load/load.go
8:	"io/ioutil"
127:		data, err := ioutil.ReadFile(path)
138:		data, err := ioutil.ReadFile(unresolvedPath)
200:		if data, err := ioutil.ReadAll(resp.Body); err != nil {
207:	data, err := ioutil.ReadAll(resp.Body)
236:		if data, err := ioutil.ReadAll(resp.Body); err != nil {
243:	resolved, err := ioutil.ReadAll(resp.Body)
277:			raw, err := ioutil.ReadFile(path)
357:				command, err := ioutil.ReadFile(filepath.Join(dir, observer.Observer.Commands))
368:				if version, err := ioutil.ReadFile(path); err == nil {
397:	command, err := ioutil.ReadFile(filepath.Join(baseDir, step.Reference.Commands
```

Reply:
There is no way for users to have literal configs in gziped format. The only case that the data are gzipped is when the files are mounted from the config maps.

Reply:
My future me when reading this:
1) Why is this using something from Prow config package when this is ci-op config?
2) Why is this place using `ReadFileMaybeGZIP` and the L138 isn't?

Reply:
1.  well that function only reads and converts the gzipped data. It has nothing to do with prow configs.
  2. Nice catch. I will fix this as well.

Reply:
updated